tests/geo/test_curve_objectives.py

```diff
@@ -358,6 +358,40 @@ def test_curve_surface_distance(self):
             assert err_new < 0.3 * err
             err = err_new
 
+    def test_linking_number(self):
+
+        curves1 = create_equally_spaced_curves(2, 1, stellsym=True, R0=1, R1=0.5, order=5, numquadpoints=128)
+
+        from simsopt.geo.curvexyzfourier import CurveXYZFourier
```

Comment:
It's best to keep imports at the top of the file, and I think this class is imported there already anyway.